### PR TITLE
Alert counter

### PR DESCRIPTION
**Pull request by anmolbabu**

### CODE REVIEW

File: tendrl/performance_monitoring/central_store/__init__.py [hunk 1]
```diff
@@ -72,10 +72,24 @@ def get_node_ids(self):
...
-        return alerts_arr
+    def get_alert_ids(self, node_id=None):
+        alert_ids = []
+        try:
+            alerts = tendrl_ns.etcd_orm.client.read(
+                '/alerting/nodes/%s' % node_id
+            )
+            for alert in alerts._children:
+                alert_ids.append(
+                    alert['key'][len('/alerting/nodes/%s' % node_id):]
```
Comment (shtripat) [inline tendrl/performance_monitoring/central_store/__init__.py]: You can use `alert['key'].split('/')[-1]` may be

**Comment by anmolbabu** [comment]
@Tendrl/tendrl-core Please review

**Comment by anmolbabu** [comment]
@nthomas-redhat @r0h4n @shtripat Please review

**Comment by r0h4n** [comment]
With reference to this PR: https://github.com/Tendrl/performance-monitoring/pull/71

Is this one still valid?

**Comment by anmolbabu** [comment]
Abandoning this as this has already been taken in as part of https://github.com/Tendrl/performance-monitoring/pull/71

**Pull request closed without merging**